### DISCUSSION TITLE
Adding Query Values to Default Alerting Template

### DISCUSSION THREAD

**Discussion by borghol** [grafana/grafana#41019] [Alerting]
With the removal of screenshot support in Grafana 8 Unified Alerting, it would probably go a long way to include the query values in the notification itself.

<img width="612" alt="Screen Shot 2021-10-28 at 8 56 59 AM" src="https://user-images.githubusercontent.com/8896043/139195798-718cd0c7-5141-41ec-94ad-6b35e1f8caa2.png">

I understand that you can create a custom template that includes these query values, but a notification is only as good as the information it provides. The current notification tells you what happened, and not how.

**Comment by pkolyvas** [comment]
We agree @borghol! Thanks for submitting this.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
